README.md

```diff
@@ -1,40 +1,53 @@
-|:----:|:----:|:----:|
+![KOF 2002](https://dl.dropboxusercontent.com/u/743491/PCSX2/KoF2002.jpg "KOF 2002")
+![Final Fantasy XII](https://dl.dropboxusercontent.com/u/743491/PCSX2/FinalFantasyXII.jpg "Final Fantasy XII")
+![Odin Sphere](https://dl.dropboxusercontent.com/u/743491/PCSX2/OdinSphere.jpg "Odin Sphere")
 
 # System Requirements
 
 ## Minimum
-* Windows/Linux OS
+* OS: Windows Vista or newer or GNU/Linux (32-bit or 64-bit)
 * CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up)
-* GPU: Any that supports Pixel Shader model 2.0, except Nvidia FX series (broken SM2.0, too slow anyway)
-* 512MB RAM (note Vista needs at least 2GB to run reliably)
+* GPU: DirectX 10 GPU or better
+* RAM: 2GB or more
 
 ## Recommended
-* Windows Vista / Windows 7 (32bit or 64bit) with the [latest DirectX](https://www.microsoft.com/en-us/download/details.aspx?id=8109)
-* CPU: Intel Core 2 Duo @ 3.2ghz or better
-* GPU: 8800gt or better (for Direct3D10 support)
-* RAM: 1GB on Linux/Windows XP, 2GB or more on Vista
+* OS: Windows 7/8/8.1/10 (64-bit) or GNU/Linux (64-bit)
+* CPU: Intel Haswell (or AMD equivalent) @ 3.2GHz or better
+* GPU: DirectX 11 GPU or greater
+* RAM: 4GB or more
+
+## Notes
+
+- PCSX2 1.4.0 is the last version to support Windows XP. Windows XP is no longer getting updates (including security-related udpates), and graphics drivers for Windows XP are older and no longer maintained.
+
+- If you have Windows Vista, make sure you have fully updated Windows and have the latest drivers to ensure maximum stability. Having a newer GPU is also recommended so you have the latest drivers.
+
+- Make sure to update your OS, drivers, and [DirectX](https://www.microsoft.com/en-us/download/details.aspx?id=8109) (if applicable) to make sure you have the best experience possible.
 
-**Note**: Because of copyright issues, and the complexity of trying to work around it, you need a BIOS dump extracted from a legitimately owned Playstation 2 console to use the emulator.
+- Because of copyright issues, and the complexity of trying to work around it, you need a BIOS dump extracted from a legitimately-owned PlayStation 2 console to use the emulator.
 
-**Note:** PCSX2 mainly takes advantage of 2 CPU cores. As of r4865 PCSX2 can now take advantage of a 3rd core using the MTVU speedhack. This can be a significant speedup on CPUs with 3+ cores, however on GS limited games (or on dual core CPUs) it may be a slowdown.
+- PCSX2 mainly takes advantage of 2 CPU cores. As of [commit r4865] (https://github.com/PCSX2/pcsx2/commit/ac9bf45) PCSX2 can now take advantage of a 3rd core using the MTVU speedhack. This can be a significant speedup on CPUs with 3+ cores, however on GS-limited games (or on dual-core CPUs) it may be a slowdown.
```

Comment:
google code just turned to ``read only`` a few months ago and all the project details have been archived. (Wiki / Issues / Commits)